### PR TITLE
Dịch "underfit-overfit_vn" - Phần 1

### LINKED ISSUE #840: Dịch "underfit-overfit_vn" - Phần 1
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_multilayer-perceptrons/underfit-overfit_vn.md#L1-L57

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by tranyenthy**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #840 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [x] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 1]
```diff
@@ -16,7 +16,11 @@ For example, imagine that we wanted to hunt for patterns among genetic markers l
 Because each person's genes identify them uniquely (ignoring identical siblings), it's possible to memorize the entire dataset.
 -->
 
-*dịch đoạn phía trên*
+Như những nhà khoa học học máy, mục tiêu của chúng ta đó là khám phá ra các *mẫu*.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]: Hmm, `khám phá mẫu` mình đọc cứ tưởng `explore sample`. Vầy thì sao nhỉ?
```suggestion
Là những nhà khoa học học máy, mục tiêu của chúng ta đó là khám phá ra các *khuôn mẫu*.
```

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 2]
```diff
@@ -16,7 +16,11 @@ For example, imagine that we wanted to hunt for patterns among genetic markers l
...
+Nhưng làm sao chúng ta có thể chắc chắn rằng chúng ta đã thực sự khám phá ra một mẫu mà nó *phổ quát* và không chỉ đơn giản là ghi nhớ các dữ liệu của chúng ta.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]:
```suggestion
Nhưng làm sao chúng ta có thể chắc chắn rằng chúng ta đã thực sự khám phá ra một khuôn mẫu *phổ quát* chứ không chỉ đơn giản là ghi nhớ dữ liệu.
```

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 3]
```diff
@@ -16,7 +16,11 @@ For example, imagine that we wanted to hunt for patterns among genetic markers l
...
+Ví dụ, thử tưởng tượng rằng chúng ta muốn săn lùng các mẫu trong số các dấu hiệu di truyền liên kết các bệnh nhân và tình trạng mất trí của họ,
+(hãy để các nhãn được trích ra từ bộ {*mất trí nhớ*, *suy giảm nhận thức mức độ nhẹ*, *khỏe mạnh*}).
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]:
```suggestion
Ví dụ, thử tưởng tượng rằng chúng ta muốn săn lùng các khuôn mẫu liên kết các dấu hiệu di truyền của bệnh nhân và tình trạng mất trí của họ, với nhãn được trích ra từ tập {*mất trí nhớ*, *suy giảm nhận thức mức độ nhẹ*, *khỏe mạnh*}).
```

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 4]
```diff
@@ -16,7 +16,11 @@ For example, imagine that we wanted to hunt for patterns among genetic markers l
...
+Bởi vì các gene của mỗi người định dạng họ theo cách độc nhất vô nhị (bỏ qua các cặp song sinh giống hệt nhau), nên hoàn toàn có thể để ghi nhớ toàn bộ tập dữ liệu.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]:
```suggestion
Bởi vì các gene của mỗi người định dạng họ theo cách độc nhất vô nhị (bỏ qua các cặp song sinh giống hệt nhau), nên việc ghi nhớ toàn bộ tập dữ liệu là hoàn toàn khả thi.
```

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 5]
```diff
@@ -25,15 +29,20 @@ When we deploy the model in the future, we will encounter patients that the mode
 Our predictions will only be useful if our model has truly discovered a *general* pattern.
 -->
 
-*dịch đoạn phía trên*
+Chúng ta không muốn mô hình của chúng ta nói rằng *"Bob kìa! Tôi nhớ anh ta! Anh ta bị mất trí nhớ!*
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]:
```suggestion
Chúng ta không muốn mô hình của nói rằng *"Bob kìa! Tôi nhớ anh ta! Anh ta bị mất trí nhớ!*
```

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 6]
```diff
@@ -25,15 +29,20 @@ When we deploy the model in the future, we will encounter patients that the mode
...
+Lý do đơn giản.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]:
```suggestion
Lý do tại sao rất đơn giản.
```

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 7]
```diff
@@ -25,15 +29,20 @@ When we deploy the model in the future, we will encounter patients that the mode
...
+Khi chúng ta triển khai mô hình trong tương lai, chúng ta sẽ chạm trán các bệnh nhân mà mô hình của chúng ta chưa bao giờ gặp trước đấy.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]:
```suggestion
Khi chúng ta triển khai mô hình trong tương lai, chúng ta sẽ gặp các bệnh nhân mà mô hình chưa bao giờ gặp trước đấy.
```

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 8]
```diff
@@ -25,15 +29,20 @@ When we deploy the model in the future, we will encounter patients that the mode
...
+Sự dự toán của chúng ta sẽ chỉ có ích khi mô hình của chúng ta thực sự khám phá ra một mẫu *phổ quát*.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]:
```suggestion
Các dự đoán sẽ chỉ có ích khi mô hình của chúng ta thực sự khám phá ra một khuôn mẫu *phổ quát*.
```

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 9]
```diff
@@ -25,15 +29,20 @@ When we deploy the model in the future, we will encounter patients that the mode
...
 <!--
 To recapitulate more formally, our goal is to discover patterns that capture regularities in the underlying population from which our training set was drawn.
 If we are successfull in this endeavor, then we could successfully assess risk even for individuals that we have never encountered before.
 This problem---how to discover patterns that *generalize*---is the fundamental problem of machine learning.
 -->
 
-*dịch đoạn phía trên*
+Để tóm tắt một cách chính thức hơn, mục tiêu của chúng ta là khám phá các mẫu mà chúng nắm bắt được các quy tắc trong tập tổng thể nền tảng mà từ đó tập huấn luyện của chúng ta đã được trích ra. 
+Nếu chúng ta thành công trong nỗ lực này, thì chúng ta có thể đánh giá thành công rủi ro ngay cả đối với các cá nhân mà chúng ta chưa bao giờ gặp phải trước đây.
+Vấn đề này---làm cách nào để khám phá ra các mẫu mà *phổ quát hóa*---là vấn đề nền tảng của học máy.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]: `phổ quát` với `khái quát` có giống nhau không mọi người, chỗ này mình không rõ lắm. tag đại diện @tranyenthy @minhduc0711 @tiepvupsu @tanduong 
Theo mình hiểu `phổ quát` giống kiểu `popularize`, còn `khái quát` mới là `generalize`.

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 10]
```diff
@@ -43,7 +52,11 @@ In a large hospital system, we might access hundreds of thousands of medical rec
 When working with finite samples, we run the risk that we might discover *apparent* associations that turn out not to hold up when we collect more data.
 -->
 
-*dịch đoạn phía trên*
+Sự nguy hiểm là khi chúng ta huấn luyện các mô hình, chúng ta truy cập chỉ một mẫu dữ liệu nhỏ.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]:
```suggestion
Nguy hiểm là khi huấn luyện các mô hình, chúng ta chỉ truy cập một tập dữ liệu nhỏ.
```

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 11]
```diff
@@ -43,7 +52,11 @@ In a large hospital system, we might access hundreds of thousands of medical rec
...
+Các tệp dữ liệu hình ảnh công cộng lớn nhất chứa khoảng một triệu hình ảnh.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]: `dữ liệu công cộng` nghe tếu vãi :))

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 12]
```diff
@@ -43,7 +52,11 @@ In a large hospital system, we might access hundreds of thousands of medical rec
...
+Thông thường hơn, chúng ta phải học chỉ từ hàng ngàn hoặc hàng chục ngàn điểm dữ liệu.
+Trong một hệ thống bệnh viện lớn, chúng ta có thể truy cập hàng trăm ngàn hồ sơ y tế.
+Khi làm việc với các mẫu hữu hạn, chúng ta gặp phải rủi ro mà chúng ta có thể khám phá các sự kết hợp *rõ ràng* mà hóa ra không giữ được khi chúng ta thu thập thêm dữ liệu.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]:
```suggestion
Khi làm việc với các tập mẫu hữu hạn, chúng ta gặp phải rủi ro sẽ khám phá ra các sự kết hợp *rõ ràng* mà hóa ra lại không áp dụng khi chúng ta thu thập thêm dữ liệu.
```

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md [hunk 13]
```diff
@@ -52,7 +65,9 @@ If you altered the model structure or the hyper-parameters during the experiment
 you might have noticed that with enough nodes, layers, and training epochs, the model can eventually reach perfect accuracy on the training set, even as the accuracy on test data deteriorates.
 -->
 
-*dịch đoạn phía trên*
+Hiện tượng việc khớp của dữ liệu huấn luyện càng chặt chẽ nhiều hơn là khi chúng ta khớp các phân phối nền tảng được gọi là quá khớp, và kỹ thuật sử dụng để chống lại quá khớp được gọi là điều chuẩn.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/underfit-overfit_vn.md]:
```suggestion
Hiện tượng việc mô hình khớp dữ liệu huấn luyện chặt chẽ hơn nhiều so với khớp phân phối nền tảng được gọi là quá khớp, và kỹ thuật sử dụng để chống lại quá khớp được gọi là điều chuẩn.
```

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md (hunk 1 above)
Comment (tranyenthy) [reply chapter_multilayer-perceptrons/underfit-overfit_vn.md]: chỗ này mình đồng ý. *khuôn mẫu* nghe thuận tai hơn là *mẫu* không

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md (hunk 11 above)
Comment (tranyenthy) [reply chapter_multilayer-perceptrons/underfit-overfit_vn.md]: > Thank bạn, bản dịch tốt rồi. Mình có nhận xét chung là bạn xài nhiều chữ `chúng ta` quá :)).
> 
> * Chữ `pattern` mình thấy dịch rõ thành `khuôn mẫu` dễ phân biệt với `sample-mẫu` hơn.
> * Chữ `phổ quát` mình không chắc có tương đương với `generalization` không?

Mình cũng băn khoăn khi dịch từ *general* thành *phổ quát*, một phần mình tham khảo trong glossary nên sử dụng. 

> `dữ liệu công cộng` nghe tếu vãi :))
Đồng ý luôn. Bạn có gợi ý gì cho mình ở chỗ này ko?

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md (hunk 9 above)
Comment (minhduc0711) [reply chapter_multilayer-perceptrons/underfit-overfit_vn.md]: mình chưa nghe từ `phổ quát` bao giờ luôn :)) nhưng có vẻ `khái quát` là mới từ đúng

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md (hunk 1 above)
Comment (minhduc0711) [reply chapter_multilayer-perceptrons/underfit-overfit_vn.md]: mình dịch từ này là `quy luật` được không nhỉ @lkhphuc @tranyenthy

Comment (minhduc0711) [reply chapter_multilayer-perceptrons/underfit-overfit_vn.md]: btw mình vừa thấy trong glossary dịch `pattern recognition` là `nhận dạng mẫu` :smile:

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md (hunk 11 above)
Comment (minhduc0711) [reply chapter_multilayer-perceptrons/underfit-overfit_vn.md]: mình suggest `tập dữ liệu công khai`

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md (hunk 1 above)
Comment (lkhphuc) [reply chapter_multilayer-perceptrons/underfit-overfit_vn.md]: `khuôn mẫu` vẫn sát hơn. `quy luật` thì có từ `rule` roài.

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md (hunk 11 above)
Comment (lkhphuc) [reply chapter_multilayer-perceptrons/underfit-overfit_vn.md]: @tranyenthy cảm ơn bạn đã thả tim gợi ý của mình. Chắc bạn chưa biết, cơ mà gợi ý nào mà bạn đồng ý thì có thể `commit` hoặc `add to batch` để commit 1 lần nhiều gợi ý luôn. Chỗ nào bạn chưa đồng ý thì cứ để đó phản biện hoặc chờ thêm ý kiến.

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md (hunk 9 above)
Comment (tranyenthy) [reply chapter_multilayer-perceptrons/underfit-overfit_vn.md]: > mình chưa nghe từ `phổ quát` bao giờ luôn :)) nhưng có vẻ `khái quát` mới là từ đúng

chỗ này mình dùng từ glossary ấy

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md (hunk 11 above)
Comment (tranyenthy) [reply chapter_multilayer-perceptrons/underfit-overfit_vn.md]: oops. okay bạn. @lkhphuc

File: chapter_multilayer-perceptrons/underfit-overfit_vn.md (hunk 9 above)
Comment (duythanhvn) [reply chapter_multilayer-perceptrons/underfit-overfit_vn.md]: * **`phổ quát`** chỉ sự rộng khắp, bao trùm.
* **`khái quát`** chỉ các vấn đề có tính chất chung, điểm chung.

Từ này không có trong Bảng thuật ngữ @tranyenthy, có được dùng tại cụm AGI nhưng chỗ đó thì dùng từ `phổ quát` là hợp lý rồi. Từ `general` phải tuỳ context.
